samples/v1.2/Scenarios/FlightDetails.json

```diff
@@ -171,47 +157,32 @@
 											"type": "Column",
 											"items": [
 												{
-													"type": "TextBlock",
-													"horizontalAlignment": "Right",
-													"text": "Sat, Mar 4, 2017",
-													"isSubtle": true,
-													"wrap": true
+													"type": "RichTextBlock",
```

Comment:
>RichTextBlock [](start = 22, length = 13)

Do we have (or plan to have) a customer facing document on how to make sure your cards are accessible? This seems like something we should capture in a tips and tricks kind of way to encourage others to avoid this pitfall.

Reply:
filed #4746